src/components/Sidebar/SidebarItem/SidebarItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Should this be a `.tsx` file? If so, we can use TS types over `prop-types`.

Reply:
That gives me the fear for some reason, I'll have a look but you are right.